packages/@react-native-windows/cli/templates/cs/src/App.xaml.cs

```diff
@@ -30,12 +30,7 @@ public App()
             Microsoft.ReactNative.Managed.AutolinkedNativeModules.RegisterAutolinkedNativeModulePackages(PackageProviders); // Includes any autolinked modules
 
             PackageProviders.Add(new Microsoft.ReactNative.Managed.ReactPackageProvider());
-{{^useExperimentalNuget}}
-            PackageProviders.Add(new Microsoft.ReactNative.Managed.ReflectionReactPackageProvider<App>());
-{{/useExperimentalNuget}}
-{{#useExperimentalNuget}}
             PackageProviders.Add(new ReactPackageProvider());
-{{/useExperimentalNuget}}
```

Comment:
so if i understand this correctly this PR is not actually making C# codegen the default - it is removing the reflection based package provider altogether.
A more accurate implementation would be to put this behind an option instead of removing it, is this something you've considered?

Reply:
I could add a flag, but I was planning to handle this in documentation. To not use the codegen output, they can update the partial class to use the reflection based class.